web/themes/custom/my_awesome_theme/templates/menu/menu--extras--social-network--footer.html.twig

```diff
@@ -10,6 +10,12 @@
   {% endif %}
 {% endfor %}
 
+{% set social_label = config_pages_field('global_configurations', 'field_social_label', 'full') %}
+{% if (social_label) %}
```

Comment:
unnecessary brackets, but whatever.
Also this check is not enough, if global configuration exist, but field is empty you will render empty h2 tag